### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by crow**
Updates the gimbal adapters for conformance with the latest Airship apis. Adds enter location event for delayed enter gimbal delegate callback.

update for release 2.0.0

### CODE REVIEW

**Review by BrianBatchelder — APPROVED** [review]
👍

**Comment by marc-scig** [comment]
I made a Podspec for this, since most of our other adapters and plugins already use Cocoapods, and updated the README/CHANGELOG for this release. It appears to all build and integrate smoothly in a test app I made. However I'm not very experienced with Gimbal, so I'm not very clear on how to test this beyond running through basic integration and setup.

File: Pod/Classes/GimbalAdapter.swift [hunk 1]
```diff
@@ -1,13 +1,20 @@
-/* Copyright 2017 Urban Airship and Contributors */
+/* Copyright Airship and Contributors */
 
-import AirshipKit
+
+#if canImport(AirshipCore)
```
Comment (rlepinski) [inline Pod/Classes/GimbalAdapter.swift]: Should just be Airship right? Since its a pod we should expect that they are using pods?

**Comment by rlepinski** [comment]
I would rename the podspec to Airship-GimbalAdapter

**Review by rlepinski — APPROVED** [review]

**Pull request merged**